backend/video/views.py

```diff
@@ -1,11 +1,12 @@
+from tracemalloc import start
+import urllib
 from datetime import timedelta
 from io import StringIO
-
 import requests
-import urllib
 import webvtt
 from drf_yasg import openapi
 from drf_yasg.utils import swagger_auto_schema
+from mutagen.mp3 import MP3
```

Comment:
Can you please add this to `requirements.txt`

Reply:
Yes yes.